external/obj-file-demo.js

```diff
@@ -0,0 +1,156 @@
+import {defs, tiny} from './common.js';
```

Comment:
did u get obj-file-demo.js from somewhere? asking bc of the nicely written paragraph style comments like in our assignment code.

also, where is this actually called?